master/pkg/tasks/task.go

```diff
@@ -228,6 +230,28 @@ func (t *TaskSpec) ToDockerSpec() cproto.Spec {
 		})
 	}
 
+	conf, hostConf, networkConf, err := dockerflags.Parse(env.DockerFlags())
+	if err != nil {
+		return cproto.Spec{}, errors.Wrap(err, "error adding docker flags to Docker config")
+	}
+
+	conf.User = getUser(t.AgentUserGroup)
+	conf.ExposedPorts = toPortSet(env.Ports())
+	conf.Env = envVars
+	conf.Cmd = t.Entrypoint
+	conf.Image = env.Image().For(deviceType)
+	conf.WorkingDir = t.WorkDir
+
+	hostConf.Privileged = false // Users setting privileged could be bad from a security standpoint.
```

Comment:
Wondering if there are any other flags that should be disallowed for potential to interfere with other tasks on an agent

Reply:
This probably needs an rbac permission, since unfortunately people wanna set this sometimes

Reply:
But yeah, this probably could use an audit.